exercises/variable-length-quantity/canonical-data.json

```diff
@@ -0,0 +1,185 @@
+{
+  "#": [
+    "JSON doesn't allow hexadecimal literals.",
+    "All numbers are given as decimal literals instead.",
+    "It is highly recommended that your track's test generator display all numbers as hexadecimal literals."
+  ],
+  "encode": {
+    "single": {
+      "description": ["Encode a single integer, producing a series of bytes."],
+      "cases": [
+        {
+          "description": "zero",
+          "input": 0,
+          "expected": [0]
+        },
+        {
+          "description": "arbitrary single byte",
+          "input": 64,
+          "expected": [64]
+        },
+        {
+          "description": "largest single byte",
+          "input": 127,
+          "expected": [127]
+        },
+        {
+          "description": "smallest double byte",
+          "input": 128,
+          "expected": [129, 0]
+        },
+        {
+          "description": "arbitrary double byte",
+          "input": 8192,
+          "expected": [192, 0]
+        },
+        {
+          "description": "largest double byte",
+          "input": 16383,
+          "expected": [255, 127]
+        },
+        {
+          "description": "smallest triple byte",
+          "input": 16384,
+          "expected": [129, 128, 0]
+        },
+        {
+          "description": "arbitrary triple byte",
+          "input": 1048576,
+          "expected": [192, 128, 0]
+        },
+        {
+          "description": "largest triple byte",
+          "input": 2097151,
+          "expected": [255, 255, 127]
+        },
+        {
+          "description": "smallest quadruple byte",
+          "input": 2097152,
+          "expected": [129, 128, 128, 0]
+        },
+        {
+          "description": "arbitrary quadruple byte",
+          "input": 134217728,
+          "expected": [192, 128, 128, 0]
+        },
+        {
+          "description": "largest quadruple byte",
+          "input": 268435455,
+          "expected": [255, 255, 255, 127]
+        },
+        {
+          "description": "smallest quintuple byte",
+          "input": 268435456,
+          "expected": [129, 128, 128, 128, 0]
+        },
+        {
+          "description": "arbitrary quintuple byte",
+          "input": 4278190080,
+          "expected": [143, 248, 128, 128, 0]
+        },
+        {
+          "description": "maximum 32-bit integer input",
+          "input": 4294967295,
+          "expected": [143, 255, 255, 255, 127]
+        }
+      ]
+    },
+    "multiple": {
```

Comment:
I reconsidered, based on a concern that "single" and "multiple" being diferent cases could mislead language implementors into believing that their test code should test `decode_single`, `decode_multiple`, etc., whereas the intention is actually exactly as you say:

> Surely single is just a multiple with size == 1?

There should only be one `decode` function tested. Let's have just the single section, to make it clear.